examples/idm/job.json

```diff
@@ -0,0 +1,25 @@
+{
+    "nevents": 10000,
+    "run_params": "2pt3",
+    "Map": 300.0,
+    "mchi": 100.0,
+    "dmchi": 10.0,
+    "seed": 879483,
+    "target_thickness": 0.002,
```

Comment:
Since we are preparing first to do this analysis for 2016 data, want to make sure it is known that the target thickness is 0.0004062 cm in 2016. More comments on 2016 conditions to come.